downstream/modules/automation-hub/proc-set-rhcertified-remote.adoc

```diff
@@ -7,6 +7,10 @@ You can edit the *rh-certified* remote repository to synchronize collections fro
 By default, your {PrivateHubName} `rh-certified` repository includes the URL for the entire group of {CertifiedName}.
 To use only those collections specified by your organization, you must include a unique URL.
```

Comment:
`To use only those collections specified by your organization, you must include a unique URL.` this refers to using the synclist url that is specific to the organization, and it brings down collections based on what the org admin toggled on the CRC server. This is the part that is being deprecated. It is still functional now, but will go away in the future (no timeline decided, but likely 2.5 or 2.6)